@navikt/core/react/src/form/form-progress/FormProgress.stories.tsx

```diff
@@ -37,6 +38,92 @@ export const Default: StoryFn<ControllableProps> = (props) => (
 );
 Default.args = { activeStep: 2, totalSteps: 7, interactiveSteps: true };
 
+export const ProvidedTranslations: StoryFn = () => {
+  const translations = {
+    FormProgress: {
+      step: "Step {activeStep} of {totalSteps}",
+      showAllSteps: "Show all steps",
+      hideAllSteps: "Hide all steps",
+    },
+  };
+
+  return (
+    <Provider translations={translations}>
+      <FormProgress activeStep={2} totalSteps={7} interactiveSteps>
+        <FormProgress.Step href="#" completed>
+          Start søknad
+        </FormProgress.Step>
+        <FormProgress.Step href="#">Personopplysninger</FormProgress.Step>
+        <FormProgress.Step interactive={false}>
+          Saksopplysninger
+        </FormProgress.Step>
+        <FormProgress.Step interactive={false}>
+          Søknadstekst for en veldig spesifikk prosess i NAV som har lang tekst
+        </FormProgress.Step>
+        <FormProgress.Step href="#">Vedlegg</FormProgress.Step>
+        <FormProgress.Step href="#">Oppsummering</FormProgress.Step>
+        <FormProgress.Step href="#">Innsending</FormProgress.Step>
+      </FormProgress>
+    </Provider>
+  );
+};
+
+export const DefaultTranslations: StoryFn = () => {
+  return (
+    <Provider>
+      <FormProgress activeStep={2} totalSteps={7} interactiveSteps>
+        <FormProgress.Step href="#" completed>
+          Start søknad
+        </FormProgress.Step>
+        <FormProgress.Step href="#">Personopplysninger</FormProgress.Step>
+        <FormProgress.Step interactive={false}>
+          Saksopplysninger
+        </FormProgress.Step>
+        <FormProgress.Step interactive={false}>
+          Søknadstekst for en veldig spesifikk prosess i NAV som har lang tekst
+        </FormProgress.Step>
+        <FormProgress.Step href="#">Vedlegg</FormProgress.Step>
+        <FormProgress.Step href="#">Oppsummering</FormProgress.Step>
+        <FormProgress.Step href="#">Innsending</FormProgress.Step>
+      </FormProgress>
+    </Provider>
+  );
+};
+
+// TODO: keep this? thoughts?
```

Comment:
This would potentially make it harder/impossible to handle contextual local overrides in components in complex applications 🤔 
Lets say i want to update some labels in an `Alert` based on local state. I don't want to hoist that state up to root for the to update translation, but rather just update it locally on the component. It feels wrong to have to wrap the local component in its own Provider just to translate, but might work better if we have our own translation component provider, something like

```
<AkselTranslation translate={{alert: {success-label: "Lorem ipsum"}}>
   <Alert />
</AkselTranslation>
``` 

Unsure if this makes it any easier since we still have to account for different languages etc here

Reply:
Yeah I agree it's a bit ugly when you have just a single component 🤔... so then we keep the component translation overrides (it gets precedence over the global ones).

I'm still not sure I understand what needs to be done to support different languages, if you want to switch a language don't the consumers simply swap out the value in the global provider then?

Reply:
On second thought, it's not _very_ ugly, I guess you wouldn't pull in Aksel components just to have a single component in your app, but probably style the entire app? (if you only want a quick component, chances might be it doesn't require translation then?) 🤔 

I guess webcomponents down the line will probably want a cleaner API (if each component is to live in isolation, they shouldn't require a wrapper for each one)

Reply:
> I'm still not sure I understand what needs to be done to support different languages, if you want to switch a language don't the consumers simply swap out the value in the global provider then?

Yes, but in the example over the Alert-component has no clue what language is currently toggled in root. One might just make it available trough a "useContext", but its quite cumbersome to have to subscribe to a context. Maybe a "WithAkselTranslation"-component could work (just braindumping here, no idea if this is a good idea)


```
<WithAkselTranslation>
    {(language)  => <Alert translation={language === "en" ? ... : .... } />}
</WithAkselTranslation>
```

Reply:
So you mean to provide all translations at once and a selection index for current one? (from the components viewpoint). I was thinking that the component would only ever see a single translation, and that would be the data to render, isn't that simpler?. Alternative is for us to implement a similar thing to this: https://www.i18next.com/overview/api#languages (language API).

Reply:
We should probably also make an example that loads multiple languages from individual language.json files. (and sends those into the providerContext somehow (though we probably want to be able to specify language on the lower level too? component level? or maybe then you are "on your own" and will do an if statement / ternary on a lang variable)